internal/cmd/asyncapi/command_import.go

```diff
@@ -288,16 +286,8 @@ func (c *command) createTopic(details *accountDetails, topicName string, kafkaBi
 	if err != nil {
 		return false, err
 	}
-	if _, httpResp, err := kafkaRest.CloudClient.CreateKafkaTopic(details.kafkaClusterId,
-		createTopicRequestData); err != nil {
-		restErr, parseErr := kafkarest.ParseOpenAPIErrorCloud(err)
-		if parseErr == nil && restErr.Code == ccloudv2.BadRequestErrorCode {
-			// Print partition limit error w/ suggestion
-			if strings.Contains(restErr.Message, "partitions will exceed") {
-				return false, errors.NewErrorWithSuggestions(restErr.Message, errors.ExceedPartitionLimitSuggestions)
-			}
-		}
```

Comment:
Are we sure we want to move all of this logic into the SDK wrappers file? IMO, the SDK wrappers should be straightforward and potentially able to be auto-generated in the future.

Reply:
I can move all suggestion wrapping back into the command files. That probably shouldn't be the job of the sdk wrappers...

But I think we can't make it completely straightforward in the ccloudv2 file, because error message replacement logic should be baked in like with `DeleteKafkaTopic`. I'm not sure how much that affects future attempts to auto generate it.